src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -1428,194 +1428,87 @@ public struct PivotPoint
         /// Top
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public static float Top
-        {
-            get
-            {
-                float ret = Interop.NDalicAnchorPoint.AnchorPointTopGet();
-                if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
-                return ret;
-            }
-        }
+        public static float Top => ParentOrigin.Top;
```

Comment:
We cannot always assuemt that `PivotPoint.Top` is same as `ParentOrigin.Top`. But for not, they are eqaully 0.0f. So looks find for these days.